opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java

```diff
@@ -22,6 +22,7 @@
 import io.opentracing.noop.NoopTracerFactory;
 import io.opentracing.propagation.Format;
 
+import java.io.IOException;
```

Comment:
Why the import of IOException?
I don't see it being thrown?

Reply:
Oh, good catch. I forgot to remove it during my cleanup.

Reply:
Will you remove the import before merging?

Reply:
Yes ;)